src/librustc_metadata/rmeta/encoder.rs

```diff
@@ -585,7 +585,7 @@ impl<'tcx> EncodeContext<'tcx> {
             println!("  def-path table bytes: {}", def_path_table_bytes);
```

Comment:
Not relevant to this PR necessarily but we should probably encode the `DefPathTable` as an actual `Table`.

Reply:
Harder to do without splitting `DefPathTable` into several tables, though.